test/Tests.Infrastructure/InterversionTest/UpgradeTestSuit.cs

```diff
@@ -127,11 +126,14 @@ public Version40X(InterversionTestBase infrastructure) : base(infrastructure)
 
         public override async Task TestClustering(List<DocumentStore> stores, string key)
         {
-            var user = new User
+            using (stores[0].SetRequestTimeout(TimeSpan.FromSeconds(60)))
```

Comment:
why does it matter?

Reply:
because the default was 15 seconds, and we might need more (as in the other implementations of `TestClustering`)